inc/widgets.php

```diff
@@ -33,6 +33,17 @@ function nightingale_wp_widgets_init() {
 		'after_title'   => '</h2>',
 	) );
 
+	// Front page sidebar (AKA secondary content area) widget area
```

Comment:
Do we still need this widget area? I thought we'd switched to just list the latest posts and not have it as widgets?

Reply:
Well spotted Chris! I have removed this code.